app/javascript/src/BackendApis/components/NewBackendForm.jsx

```diff
@@ -23,8 +23,9 @@ type Props = {
 }
 
 const VALIDATION_CONSTRAINTS = {
-  name: { length: { minimum: 1 } },
-  privateEndpoint: { url: { schemes: ['http', 'https', 'ws', 'wss'] } }
+  name: { presence: { allowEmpty: false } },
+  systemName: { format: { pattern: '^[a-z0-9-_]*$', flags: 'i' } },
```

Comment:
Are you sure this is the same pattern being validated in rails?

Reply:
No.. but unless I'm mistaken it matches the same rules: only ASCII letters, numbers, dashes and underscores.

Reply:
In the end I copied the same regex to avoid any possible issues.